src/routes/files/+page.svelte

```diff
@@ -535,21 +603,23 @@
     on:drop={e => {
         dragDrop(e)
     }}>
-    {#if isDraggingFromLocal || isFadingOutDragDropOverlay}
+    <!-- {#if isDraggingFromLocal || isFadingOutDragDropOverlay}
         <div class="overlay {isFadingOutDragDropOverlay ? 'fade-out' : ''}">
             <div class="upload-box">
                 <p>{filesCount > 1 ? $_("files.dragging_files").replace("{count}", filesCount.toString()) : $_("files.dragging_file")}</p>
             </div>
         </div>
-    {/if}
+    {/if} -->
```

Comment:
Should we go on and remove this or will this be uncommented later?

Reply:
This is some code that will be coming in from Lucas' future pr, we just commented out to hopefully make is merge easier